Cargo.toml

```diff
@@ -15,6 +15,8 @@ members = [
     "crates/conjure_macros",
     "solvers/kissat",
     "solvers/minion",
+    "tree-sitter/parser",
```

Comment:
the files should end up under `crates/tree-sitter-essence` eventually, as we discussed yesterday.